pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -304,4 +305,7 @@ def _reorder_callbacks(callbacks: List[Callback]) -> List[Callback]:
         """
```

Comment:
the docstring of the method does not match anymore

Reply:
good catch! updated!